proposed/2023/PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,45 @@
+# Markdown Readme Rendering
+* Start Date: 6/27/23
+* Authors: Allie Barry ([albarry4](https://github.com/albarry4))
+* Issue: https://github.com/NuGet/Home/issues/12583
+
+## Summary
+
+Support rendering markdown readmes in the package details page for NuGet in Visual Studio to make package affordances more accesible for customers browsing packages within the PMUI, and to increase motivation for high quality Readme inclusion for package authors. This will be part of a larger effort to enhance the experience around Readmes and increase their inclusion rate across packages in the NuGet ecosystem. 
+
+## Motivation
+
+In the latest NuGet in Visual Studio 17.5 customer survey, our team found that one of the largest problems that our customers face while using NuGet in the Visual Studio ecosystem is around understanding package affordances – specifically calling out the usage of package metadata such as READMEs. Only 35.35% of survey respondents feel that they can adequately understand the affordances of a package (does it do what I need to to do when I need it?) while browsing and installing packages. Including a detailed README file and implementing features to facilitate README viewing and inclusion would positively impact this number.   
+
+## Explanation
+
+### Functional Explanation
+
+The goal of this new experience would be to integrate the experience of viewing a rendered Readme directly into NuGet in Visual Studio. Additionally, we want to draw more attention to the README in general, and make it feel like an important and crucial element of a package.  
+
+The desired experience would somewhat mirror the user interface on NuGet.org, which already supports rendered Markdown viewing. When a customer views the package detail page of a package with an included markdown readme, they will see this proposed UI mock ![Mock](https://github.com/NuGet/Home/assets/89422562/0017703e-2572-4c3c-8f1c-7cb72a01f25f)
```

Comment:
I'm guessing the proposed changes are just the details tab correct?

Reply:
That mock is dense with features that don't exist yet. How much of it is included in this readme spec?

Reply:
Hi! The proposed changes here are just the details tab and the ability to render markdown READMEs directly in the package details pane. Any features in the mock that dont exist yet can be ignored for the time being -- but the general layout in the mock showing the rendered README in the details tab is what is being proposed here!

Reply:
If that's the case, it wouldn't be worst idea to crop the image to include only that part. 
The overall screenshot might send some mixed signals, and suggest something we're not committing to/planning right now.

Reply:
I'd ideally want more than just cropping the image, because that leaves the question of how will we render *just* the Readme in the UI we have today? 

1. Will we split the details pane into 2 columns (left is Readme, right is Package metadata/Details) 
2. Will it be the 1st new tab in the details pane? This implies that existing details are in a new tab (would produce 2 total tabs: the existing content in tab1, and the readme in tab2)
3. Will we just render Readme content below the package description?
4. other?

Reply:
@nkolev92 @donnie-msft, I just created and updated the spec with a full new mock that doesn't have any features that don't exist. This new mock should address Donnie's comment above!

Reply:
There's a new screenshot with tabs, so that aligns with my option # 2, but it shows a total of 5 tabs. Just wanted confirmation that this spec should result in all 5 tabs? 
![image](https://github.com/NuGet/Home/assets/49205731/8a64bd70-4c7c-44ab-8e2a-a38886621452)

I think I'm clear on how these tabs would work: 
✅ README
✅ Frameworks
✅ Dependencies

But I think these require work outside the scope of the spec:
⚠ Used By
⚠ Versions

Reply:
I've updated the mock again to just show those three tabs! Apologies, I missed editing out that piece as I was creating the new mock

Reply:
With the updated screenshot, it's not clear where the existing package details will be. For example, license, version, authors, data published, project URL, report abuse link.

Reply:
In Visual Studio, I also we should be mindful adding a lot of new tabs. 
Today most of that information is within 1 pane (license, author, url, dependencies etc), we should keep that.

IMO it should be `README` and `Package Details` unless we have a good reason to believe we need the extra tabs.

Reply:
Also, the `frameworks` tab is a whole major feature itself, so unless we want that now (we don't afaik), we should not include it.